pkg/controlplane/apiserver/config.go

```diff
@@ -159,6 +163,18 @@ func BuildGenericConfig(
 		genericConfig.FlowControl, lastErr = BuildPriorityAndFairness(s, clientgoExternalClient, versionedInformers)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.UnknownVersionInteroperabilityProxy) {
+		genericConfig.PeerEndpointLeaseReconciler, lastErr = createPeerEndpointLeaseReconciler(*genericConfig, storageFactory)
+		if lastErr != nil {
+			return
+		}
+		genericConfig.PeerProxy, lastErr = BuildPeerProxy(versionedInformers, genericConfig.StorageVersionManager, s.ProxyClientCertFile,
+			s.ProxyClientKeyFile, s.GenericServerRunOptions.PeerCAFile, s.GenericServerRunOptions.PeerAdvertiseAddress)
```

Comment:
Added check for required cert files [here](https://github.com/kubernetes/kubernetes/pull/117740/files#diff-ffc0e71781b2313b01ec857839f8ff00bc36488f22892b045dbda0949960683cR225-R242), if any of these are missing, server start up will fail. Since this is feature gated, would this still count as a  breaking change? If so, I will make it a no-op.

Reply:
Gotcha, done.